### DISCUSSION TITLE
What is the open source license used in this project?

### DISCUSSION THREAD

**Discussion by piggo** [shridhar-tl/jira-assistant#297] [Q&A]
Hello, this project just states that it is 'free and open source software'. But it doesn't state what the license is. Can you please clarify the exact license type? (e.g. MIT, Apache 2.0, GPL, etc?)

**Comment by shridhar-tl** [comment]
I have never thought much about licensing as such. So at this point no specific license is attached. Extensions follow web stores default licensing. While the source code would follow GPL. I will try and put something in. But until then you can consider it as GPL.

Hope that helps. Please feel free to get back to me in case of anything.
